wrapper/Convert/SireOpenMM/sire_to_openmm_system.cpp

```diff
@@ -400,6 +401,111 @@ void _add_positional_restraints(const SireMM::PositionalRestraints &restraints,
     }
 }
 
+void _add_rmsd_restraints(const SireMM::RMSDRestraints &restraints,
+                              OpenMM::System &system, LambdaLever &lambda_lever,
+                              int natoms)
+{
+    if (restraints.isEmpty())
+        return;
+    
+    // Internal functions to convert to OpenMM units
+    const double internal_to_nm = (1 * SireUnits::angstrom).to(SireUnits::nanometer);
+    const double internal_to_k = (1 * SireUnits::kcal_per_mol / (SireUnits::angstrom2)).to(SireUnits::kJ_per_mol / (SireUnits::nanometer2));
+
+    // Function to covnert SireMaths::Vector to OpenMM::Vec3 
+    auto to_vec3 = [](const SireMaths::Vector &coords)
+    {
+        const double internal_to_nm = (1 * SireUnits::angstrom).to(SireUnits::nanometer);
+
+        return OpenMM::Vec3(internal_to_nm * coords.x(),
+                            internal_to_nm * coords.y(),
+                            internal_to_nm * coords.z());
+    };
+
+    // Count the number of existing RMSD forces in the system
+    std::vector<OpenMM::Force*> forces;
+    
+    for (int i = 0; i < system.getNumForces(); ++i) {
+        forces.push_back(&system.getForce(i));
+    }
+
+    int n_CVForces = 0;
+
+    for (auto* force : forces) {
+        if (dynamic_cast<OpenMM::CustomCVForce*>(force)) {
+            n_CVForces++;
+        }
+    }
+
+    // Apply RMSD restraining potential using OpenMM::CustomCVForce
+    const auto atom_restraints = restraints.restraints();
+    for (const auto &restraint : atom_restraints)
+    {
+        // Define unique parameter names for rho, k and rmsd_b
+        // std::string rho_unique = "rho_" + std::to_string(n_CVForces);
+        std::string rho_unique = "rho_" + std::to_string(n_CVForces);
+        std::string k_unique = "k_" + std::to_string(n_CVForces);
+        std::string rmsd_b_unique = "rmsd_b_" + std::to_string(n_CVForces);
+        // Unique CV name
+        std::string rmsd_unique = "rmsd_" + std::to_string(n_CVForces);
+          
+        // Single parameter naming
+        // std::string rho_unique = "rho"
+        // std::string k_unique = "k";
+        // std::string rmsd_b_unique = "rb";
+        // std::string rmsd_unique = "rmsd";
```

Comment:
Please remove this if it's no longer needed. I assume this was the initial implementation (single RMD restraint) which was superseded by the multi restraint implementation.